projects/storefrontlib/src/cms-components/misc/icon/icon.component.ts

```diff
@@ -37,7 +39,7 @@ export class IconComponent {
    * Indicates whether the icon is configured to use SVG or not.
    */
   get useSvg(): boolean {
```

Comment:
what about we make this:

```
useSvg(type: ICON_TYPE): boolean {
    return this.iconLoader.useSvg(type);
}
```

And pass in the type from the UI? 
same for `getSvgPath`.